apps/backoffice-v2/src/common/components/organisms/Combobox/Combobox.tsx

```diff
@@ -0,0 +1,106 @@
+'use client';
```

Comment:
We have this line in a few components. Is it needed? If so, when exactly?

Reply:
Already removed just not pushed. It is needed in the context of NextJS projects when the component interacts with the DOM or requires usage of `useEffect`. Not 100% on the `useEffect` bit.